### ISSUE TITLE
SkyScanner Live Flights API down?

### ISSUE THREAD

**Issue by seeARMS** [jameshopkins/skyscanner-liveflights-api#4]
See:

https://support.business.skyscanner.net/hc/en-us/articles/212682245-Why-do-I-get-an-error-when-accessing-the-Flights-Live-Pricing-API-

and

https://support.business.skyscanner.net/hc/en-us/articles/206800359-Which-services-can-I-access-and-what-are-the-rate-limits-

**Comment by jameshopkins** [comment]
Hey,

What issue(s) are you experiencing? Is the API returning an error that is difficult to decipher, or are you asking if the underlying REST API is down? If it's the latter, I'm afraid I can't help :)

**Comment by mytuny** [comment]
Yup, SkyScanner LivePrice API is down and it seems that they are doing a huge upgrade to their APIs. Here is what they say:

> Please note that access to our Hotels API and Flights Live Pricing service are currently suspended. Unfortunately we are unable to give a date for when these services will be re-launched, but please check back regularly for updates.<

Reference: https://support.business.skyscanner.net/hc/en-us/articles/206800359-Which-services-can-I-access-and-what-are-the-rate-limits-

**Comment by jameshopkins** [comment]
> SkyScanner LivePrice API is down
That's 💩
